### PR TITLE
Fix line break errors in CLI generator

### PR DESCRIPTION
**Pull request by brewn**
I don't know anything about js, so maybe there's a better way to do this? 
tenor-155273766

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/docs/habitat-cli.html.md [hunk 1]
```diff
@@ -461,7 +461,7 @@ hab cli setup
 
 ## hab config
 
-Commands relating to Habitat runtime config
+[1;33m» Installing core/hab-butterfly/0.55.0 from channel 'stable'[0m
```
Comment (brewn) [inline www/source/docs/habitat-cli.html.md]: Looks like I'm substituting one error for another right now..ugh.

**Comment by cnunciato** [comment]
Sorry for the delay on this. I finally have some time to pick it back up today.

**Comment by mgamini** [comment]
@cnunciato any updates on this?

**Comment by cnunciato** [comment]
@mgamini Ah yeah, this was fixed in #5335.

**Pull request closed without merging**